### ISSUE TITLE
1.18.2 Allow custom reprocessor recipes

### ISSUE THREAD

**Issue by Raidobw2** [BlakeBr0/MysticalCustomization#26]
**Is your feature request related to a problem? Please describe.**
Yes. I am trying to buff reprocessor outputs for all Mystical Agriculture and Mystical Customization crops. It does work for any crop from the base mod (and I can even remove its recipes), but for Mystical Customization, it doesn't seem possible.

**Describe the solution you'd like**
Either a configuration with the Mystical Customization crops JSON files, using https://github.com/BlakeBr0/MysticalAgriculture/blob/053b62347c694bb590ea2b7c5a17a221c4c692ba/src/main/java/com/blakebr0/mysticalagriculture/api/lib/PluginConfig.java if that can be useful. This would allow disabling the reprocessor recipe, which I can then readd without having a conflict for the Reprocessor machine block.
Or that the recipes' load order be adjusted if that is possible, since it appears Mystical Customization is always loaded last, which overrides anything that was meant to override it at first.

**Describe alternatives you've considered**
I tried with KubeJS, with a normal datapack, and with a datapack loader such as Paxi. Possibly I'm doing something wrong, but I'm unsure.

**Additional context**
For example, runewood seeds:
[runewood.txt](https://github.com/BlakeBr0/MysticalCustomization/files/8954341/runewood.txt)
I tried through JSON at:
data/mysticalagriculture/recipes/seed/reprocessor/runewood.json
data/mysticalagriculture/recipes/runewood_seeds_reprocessor
data/mysticalcustomization/recipes/runewood_seeds_reprocessor

The one that hit the correct location was:
mysticalagriculture/recipes/runewood_seeds_reprocessor

Although Mystical Customization's recipe was always loaded on top, which doesn't result in 2 recipes showing, but the default one like this:
![image](https://user-images.githubusercontent.com/17010632/174942662-8ad0423a-23f3-40ae-a20f-f226e75e95ad.png)
In short, it was never possible to increase the amount of output in the reprocessor (only for Mystical Customization crops). Possibly that information can be described on the wiki as well if you want, if this feature request goes through. I did read all the wiki before and it was very useful, thank you 🙂 

Thanks for considering it!

**Comment by BlakeBr0** [comment]
I'm adding the option to disable the auto generated recipes for a crop in the next version. Note that you will need Mystical Agriculture 5.1.1 for the options to function properly.

https://blakesmods.com/docs/mysticalcustomization/adding-crops#recipe-config

**Comment by Raidobw2** [comment]
That is awesome, thanks so much! 😃
